ingestion/src/metadata/ingestion/source/dashboard/metabase/client.py

```diff
@@ -77,7 +77,7 @@ def __init__(
         self.config = config
         session_token = self._get_metabase_session()
         client_config: ClientConfig = ClientConfig(
-            base_url=str(self.config.hostPort),
+            base_url=str(self.config.hostPort)[:-1],  # remove trailing slash
```

Comment:
Small nitpick (Probably for future mini refactor):

We could have a function called `remove_trailing_slash` or something like that since we are doing it in many places and it'd be self describing vs commenting everywhere

Reply:
ye agree